src/composables/ui.ts

```diff
@@ -45,7 +45,7 @@ const hiddenCards = useStorageRef<string[]>(
 );
 const otherSettings = useStorageRef<IOtherSettings>(
   {
-    secureLoginTimeout: IS_MOBILE_APP ? AUTHENTICATION_TIMEOUTS[0] : AUTHENTICATION_TIMEOUTS[5],
+    secureLoginTimeout: IS_MOBILE_APP ? AUTHENTICATION_TIMEOUTS[0] : AUTHENTICATION_TIMEOUTS[2],
```

Comment:
This piece of code is repeated many times:
```
IS_MOBILE_APP ? AUTHENTICATION_TIMEOUTS[0] : AUTHENTICATION_TIMEOUTS[2]
```

I suggest to create such global constant:
```
export const  AUTHENTICATION_TIMEOUT_DEFAULT = (IS_MOBILE_APP)
  ? AUTHENTICATION_TIMEOUTS[0]
  : AUTHENTICATION_TIMEOUTS[2];
```

And reuse it.